### PR TITLE
test(e2e,playwright): FourOhFour component

### PR DESCRIPTION
**Pull request by ArjunSingh-Rawat**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

resolves "UI Components/FourOhFour" https://github.com/freeCodeCamp/freeCodeCamp/issues/51705

camperbot cant code

### CODE REVIEW

File: e2e/four-oh-four.spec.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  test('should display 404 heres-a-quote sentence', async () => {
+    const heresAquote = page.getByTestId('404-heres-a-quote');
+    await expect(heresAquote).toBeVisible();
+    await expect(heresAquote).toContainText(
+      translations['404']['heres-a-quote']
+    );
+  });
+
+  test('should display 404 quote', async () => {
```
Comment (naomi-lgbt) [inline e2e/four-oh-four.spec.ts:39]:
```suggestion
  test('should display 404 quote', async ({ page }) => {
```

File: e2e/four-oh-four.spec.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    await expect(page.getByTestId('404-quote')).toBeVisible();
+  });
+
+  test('should display curriculum link', async () => {
```
Comment (naomi-lgbt) [inline e2e/four-oh-four.spec.ts:43]:
```suggestion
  test('should display curriculum link', async ({ page }) => {
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]
Looks good. You'll need to update this branch to the latest version of main for the tests to pass.

**Comment by ArjunSingh-Rawat** [comment]
Hi, @naomi-lgbt I am new to contributing and this is my first pr, should i commit again with updated brach

**Comment by naomi-lgbt** [comment]
Yes, you'll want to fetch the upstream `main`, merge it into your branch, and push the merge commit.

If you have questions or get stuck, feel free to ask in our [discord](https://discord.gg/PRyKn3Vbay).

**Review by huyenltnguyen — APPROVED** [review]

**Pull request closed without merging**